### ISSUE TITLE
Missing run-time check for correct number of arguments for display() and error()

### ISSUE THREAD

**Issue by btzy** [source-academy/js-slang#733]
Both the following calls should be an error according to the spec, but are currently not an error:

```
display();
display(1,"T",3);
```

It is a problem (and perhaps confusing for students) because:
- It does not follow the spec
- It hints that missing arguments turn into `undefined`, which is not taught anywhere and not mentioned anywhere in the spec
- Source-defined functions and other stdlib functions are checked as per #346, so not checking display() and error() is even more surprising

**Comment by dtlay** [comment]
could you help to review pr #732? it would be easy to do a length check of the arguments with the newer implementation. thanks!

**Comment by martin-henz** [comment]
Yes, error() and display() give runtime error, but passing more than two arguments leads to silently ignoring the extra arguments. Should be an easy fix.

**Comment by RichDom2185** [comment]
Perhaps beyond just a runtime check, we also want to support type-checking for typed variants (see #1398).
